app/src/utils/validate-item.ts

```diff
@@ -22,6 +25,13 @@ export function validateItem(item: Record<string, any>, fields: Field[], isNew:
 			});
 		}
 
+		const relation = relationsStore.getRelationsForField(field.collection, field.field);
+
+		// Check if we are dealing with a relational field that has an empty array as its value
+		if (relation.length > 0 && Array.isArray(item[field.field]) && isEmpty(item[field.field])) {
+			item[field.field] = null;
```

Comment:
Does this alter the originally passed in `item`? That might cause issues elsewhere. I think we might have to do a `cloneDeep` at the top of this function before this runs to make sure we don't alter the passed in object.

Also, mind adding some unit tests while we're at it? 🙂

Reply:
Improved and added a test to it.